src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -43,6 +43,12 @@ def field_alias_converter(self) -> Mapping[str, Callable[[str], SelectType]]:
             constants.DEVICE_CLASS_ALIAS: lambda alias: field_aliases.resolve_device_class(
                 self.builder, alias
             ),
+            constants.PROJECT_ALIAS: lambda alias: field_aliases.resolve_project_slug_alias(
+                self.builder, alias
+            ),
+            constants.PROJECT_SLUG_ALIAS: lambda alias: field_aliases.resolve_project_slug_alias(
+                self.builder, alias
+            ),
```

Comment:
Any issues duplicating this here? Or better to map "project" -> "project.slug" and use one alias function

Reply:
Nah this is completely fine :+1: